src/cmd/support_archive/resources_test.go

```diff
@@ -211,28 +211,28 @@ func TestManifestCollectionFails(t *testing.T) {
 		Return(assert.AnError)
 
 	mockedApiReader.
-		On("List", toFlatInterfaceSlice(context, getExpectedObjectsList(queries[3].groupVersionKind), queries[2].filters)...).
+		On("List", toFlatInterfaceSlice(context, getExpectedObjectsList(queries[3].groupVersionKind), queries[3].filters)...).
```

Comment:
couldn't you just iterate over `queries` and mock `mockedApiReader.On(...)` for each of the queries?

Reply:
Mocked return values for `query[1]` and `query[2]` are different than the rest. Could iterate 3-8.